src/main/java/lotto/domain/LottoRank.java

```diff
@@ -0,0 +1,36 @@
+package lotto.domain;
+
+import java.util.Arrays;
+
+public enum LottoRank {
+    FIFTH(3, 5_000), FOURTH(4, 50_000),
+    THIRD(5, 1_500_000), SECOND(5.5, 30_000_000),
+    FIRST(6, 200_000_000), NONE(0, 0);
+
+    double matchingCount;
+    int prizeMoney;
```

Comment:
```suggestion
    private final double matchingCount;
    private final int prizeMoney;
```

접근제어자를 생략한 것은 의도된 내용일까요?

Reply:
수정하였습니다!